P1/style.css

```diff
@@ -0,0 +1,176 @@
+BODY {
+    background: linear-gradient(90deg, rgb(44,44,44) 50%, rgb(50,50,50) 50%,rgb(50,50,50) 55%, rgb(40,40,40) 55%);
+    background-size: 40px 40px;
+    background-color: black;
+    padding-top: 39px;
+    font-family: "Source Sans Pro";
+    /*letter-spacing: 1px;*/
+}
+
+div {
+    padding-top: 21px;
+    padding-bottom: 21px;
+}
+
+h1{
+    color: #373435;
+    margin-top: 12px;
+    margin-bottom: 7px;
+    letter-spacing: 3px;
+}
+
+p.main-text {
+    color: #817e7f;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    font-size: 19px;
+    letter-spacing: 1px;
+    line-height: 1.2;
+}
+
+footer {
+    color: #c5c5c5;
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+div.read-more {
+    padding: 0px;
+    width: 155px;
+    position: relative;
+    margin-top: -31px;
+    padding-bottom: 20px;
```

Comment:
overrides padding above from the same class `padding: 0px;`. can be written in one line